app/grandchallenge/cases/views.py

```diff
@@ -159,3 +176,81 @@ def get_object(self):
             raise Http404
 
         return img
+
+
+class ImageWidgetSelectView(LoginRequiredMixin, View):
+    def get(self, request, *args, **kwargs):
+        query_params = request.GET.copy()
+        interface = query_params.pop("interface", None)[0]
+        widget_name = query_params.pop("WidgetChoice-" + interface, None)[0]
+        help_text = query_params.pop("help_text", None)
+
+        if widget_name:
+            if widget_name == WidgetChoices.IMAGE_SEARCH.name:
```

Comment:
Shouldn't be a need for nested if statements here, `not widget_name` should be caught by L223.